packages/core/nuxt-theme-module/theme/pages/Category.vue

```diff
@@ -17,7 +17,7 @@
         >
           <SfIcon
             size="18px"
-            color="#BEBFC4"
+            color="#43464E"
```

Comment:
@justyna-13 its worth adding string-based colors here like `primary` to storefront UI :)